src/lib/functions/registry.ts

```diff
@@ -348,43 +343,42 @@ export class FunctionsRegistry {
 
     if (event === 'loaded') {
       const icon = warningsText ? NETLIFYDEVWARN : NETLIFYDEVLOG
-      const color = warningsText ? chalk.yellow : chalk.green
+      const color = warningsText ? picocolors.yellow : picocolors.green
       const mode =
-        // @ts-expect-error TS(2339) FIXME: Property 'logLambdaCompat' does not exist on type ... Remove this comment to see the full error message
         func?.runtimeAPIVersion === 1 && this.logLambdaCompat
```

Comment:
This was actually a legit bug. `logLambdaCompat` was a static field so this was always falsy.